client/src/components/AddButton.js

```diff
@@ -13,8 +13,7 @@ export const AddButton = styled.button`
   background-position: center;
   box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.4);
   &:focus {
-    outline: ${colors.systemAction};
-    box-shadow: 0 0 3px 2px ${colors.systemAction};
+    outline: none;
```

Comment:
I think this belongs to [this PR](https://github.com/berlinum/bricks/pull/23/commits/d371a643468e98ebb0bee581caba4add04bdb450)